src/types/types.ts

```diff
@@ -1,8 +1,12 @@
 export interface Country {
-  name: string;
+  name: {
+    common: string;
+  };
   population: number;
   region: string;
```

Comment:
Region jest czymś znanym i nie jest to każdy string a jedna z kilku z góry opisanych wartości. W takich przypadkach warto skorzystać albo z union type albo enuma do opisania tego typu (w ramach ćwiczenia niech będzie to enum)